modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -1,48 +1,27 @@
 = Create a Serverless Cluster
 :description: Learn how to create a Serverless cluster.
 :page-cloud: true
-:page-beta: true
 
-Redpanda Serverless is the fastest and easiest way to start event streaming in the cloud. 
+To start using Redpanda Serverless, sign up for a free trial. If you want to continue after the trial ends, you can enter a credit card and pay as you go. 
 
-== What is Redpanda Serverless?
+* To ask questions about the trial, use the *#serverless* https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
+* For information about Redpanda Cloud cluster types, see xref:./cloud-overview.adoc[].
+* For information about billing after the trial ends, see xref:deploy:deployment-option/cloud/manage-billing/billing.adoc[].
 
-Redpanda Serverless is a multi-tenant, globally-distributed managed streaming service (Kafka API) that abstracts users from scaling and operational concerns. It applies usage-based billing, so you only pay for what you consume. Serverless clusters provide data isolation and data residency:
+== Prerequisites
 
-* Your data is hosted on Redpanda Cloud clusters shared among multiple tenants. Other tenants never access your data.
-* Your data is stored in the Redpanda Cloud account, in the region you select. Data never leaves that region.
+. https://redpanda.com/try-redpanda/cloud-trial#serverless-trial[Sign up for a trial^] of Redpanda Serverless. 
 
-=== Serverless vs Dedicated and BYOC
-
-Redpanda Serverless is a good fit for the following:
-
-* Starter and growing workloads
-* Spiky workloads (development environments, systems that only occasionally get busy, or workloads that come and go)
-* Fast and dynamic cluster creation: you can use a Serverless cluster as an isolated container for topics
-
-Because a Serverless cluster is created in under one second, you can surface it in your applications (for example, for tenant isolation).
-
-If necessary, workloads can be migrated from Redpanda Serverless to Redpanda Dedicated or BYOC clusters. Consider Dedicated or BYOC for the following:
-
-* Control over cluster settings
-* Control over the deployment
-* Private networking
-* Workloads with consistently-high throughput
-
-=== Deployment and networking
-
-Redpanda Serverless currently is available in certain AWS regions. We expect your applications to be deployed in the same AWS region. For best performance, select the region closest to your applications. Serverless is not guaranteed to be pinned to a particular availability zone within that region.
-
-Your applications talk to Serverless over the public internet using encrypted connections. Serverless does not support private networking (that is, VPC peering to talk over private IPs). Use Redpanda Dedicated clusters for private networking.
+. Make sure you have the latest version of `rpk`. See xref:get-started:rpk-install.adoc[].
```

Comment:
```suggestion
. Latest version of `rpk`. See xref:get-started:rpk-install.adoc[].
```

Reply:
leaving this so this 2nd prereq has parallel structure with the 1st prereq